CPAC/pipeline/engine.py

```diff
@@ -2043,9 +2045,30 @@ def ingress_freesurfer(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id):
                 creds_path=data_paths["creds_path"],
                 dl_dir=cfg.pipeline_setup["working_directory"]["path"],
             )
-            rpool.set_data(
-                key, fs_ingress, "outputspec.data", {}, "", f"fs_{key}_ingress"
-            )
+            # if .mgz reorient to RPI
+            if outfile.endswith(".mgz"):
+                reorient_mgz = pe.Node(
+                    Function(
+                        input_names=["in_file", "orientation", "out_file"],
+                        output_names=["out_file"],
+                        function=mri_convert_reorient,
+                    ),
+                    name=f"reorient_mgz_{key}",
+                )
+                # Flip orientation before reorient because mri_convert's orientation is opposite that of AFNI
+                reorient_mgz.inputs.orientation = flip_orientation_code(
+                    cfg.pipeline_setup["desired_orientation"]
+                )
```

Comment:
Whoa, opposite in all 3 dimensions 🤯

Reply:
Link to the conversation on Slack (for posterity and those with access): https://cmi-cnl.slack.com/archives/C059AV12P63/p1742849534055409